src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -1545,10 +1545,10 @@ static void window_tile_inspector_invalidate(rct_window *w)
         w->widgets[WIDX_SCENERY_CHECK_QUARTER_W].top = GBBT(propertiesAnchor, 1) - 5 + 7 * 1;
         w->widgets[WIDX_SCENERY_CHECK_QUARTER_W].bottom = w->widgets[WIDX_SCENERY_CHECK_QUARTER_W].top + 13;
         // This gets the relative rotation, by subtracting the camera's rotation, and wrapping it between 0-3 inclusive
-        bool N = (tileElement->type & TILE_ELEMENT_QUADRANT_MASK) == ((0 - get_current_rotation()) & 3) << 6;
-        bool E = (tileElement->type & TILE_ELEMENT_QUADRANT_MASK) == ((1 - get_current_rotation()) & 3) << 6;
-        bool S = (tileElement->type & TILE_ELEMENT_QUADRANT_MASK) == ((2 - get_current_rotation()) & 3) << 6;
-        bool W = (tileElement->type & TILE_ELEMENT_QUADRANT_MASK) == ((3 - get_current_rotation()) & 3) << 6;
+        bool N = tileElement->GetSceneryQuadrant() == ((0 - get_current_rotation()) & 3);
+        bool E = tileElement->GetSceneryQuadrant() == ((1 - get_current_rotation()) & 3);
+        bool S = tileElement->GetSceneryQuadrant() == ((2 - get_current_rotation()) & 3);
+        bool W = tileElement->GetSceneryQuadrant() == ((3 - get_current_rotation()) & 3);
```

Comment:
The outer parentheses are superfluous now.

Reply:
I think it's clearer this way.